project/src/components/hotel-card.tsx

```diff
@@ -1,35 +1,49 @@
-const HotelCard = (): JSX.Element => (
-  <article className="cities__card place-card">
-    <div className="cities__image-wrapper place-card__image-wrapper">
-      <a href="/#">
-        <img className="place-card__image" src="img/room.jpg" width="260" height="200" alt="Place" />
-      </a>
-    </div>
-    <div className="place-card__info">
-      <div className="place-card__price-wrapper">
-        <div className="place-card__price">
-          <b className="place-card__price-value">&euro;80</b>
-          <span className="place-card__price-text">&#47;&nbsp;night</span>
-        </div>
-        <button className="place-card__bookmark-button place-card__bookmark-button--active button" type="button">
-          <svg className="place-card__bookmark-icon" width="18" height="19">
-            <use xlinkHref="#icon-bookmark"></use>
-          </svg>
-          <span className="visually-hidden">In bookmarks</span>
-        </button>
+import { Link } from 'react-router-dom';
+import {Offer} from '../types/offer';
+
+type HotelCardProps = {
+  offer: Offer;
+  handlerFocusOffer:(id: number)=>void;
+  clearOfferId:()=>void;
```

Comment:
Не забываем про пробелы)
```suggestion
  handlerFocusOffer: (id: number) => void;
  clearOfferId: () => void;
```